Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM adoptopenjdk/openjdk11:alpine-slim
```

Comment:
Adopt JDK series is deprecated. Can we use debug distroless images from Google?

Reply:
Ok. Will check out the correct version to use here.

Reply:
the issue is gcr.io/distroless/java:11-debug is not containing /bin/sh and docker is throwing errors while trying to start the service